apps/prairielearn/elements/pl-checkbox/pl-checkbox.py

```diff
@@ -712,31 +766,29 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
     submitted_set = set(submitted_keys)
     correct_set = set(correct_keys)
 
-    score = 0
-    if not partial_credit and submitted_set == correct_set:
-        score = 1
-    elif partial_credit:
-        if partial_credit_method == "PC":
-            if submitted_set == correct_set:
-                score = 1
-            else:
-                n_correct_answers = len(correct_set) - len(correct_set - submitted_set)
-                points = n_correct_answers - len(submitted_set - correct_set)
-                score = max(0, points / len(correct_set))
-        elif partial_credit_method == "EDC":
-            number_wrong = len(submitted_set - correct_set) + len(
-                correct_set - submitted_set
-            )
-            score = 1 - 1.0 * number_wrong / number_answers
-        elif partial_credit_method == "COV":
-            n_correct_answers = len(correct_set & submitted_set)
-            base_score = n_correct_answers / len(correct_set)
-            guessing_factor = n_correct_answers / len(submitted_set)
-            score = base_score * guessing_factor
+    # NOTE should probably compute this in a separate function for easier testing
```

Comment:
Why not do this now? If not, mark this `TODO` not `NOTE`

Reply:
I think it's more useful to make this change along with something else that also modifies the grading algorithm. Happy to make it into a todo but small enough change it's probably better to bundle with something else that will benefit from this.